plugins/UsersManager/API.php

```diff
@@ -988,6 +997,13 @@ public function deleteUser($userLogin)
         $this->model->deleteUserOptions($userLogin);
         $this->model->deleteUserAccess($userLogin);
 
+        $container = StaticContainer::getContainer();
+        $email = $container->make(UserDeletedEmail::class, array(
+            'login' => Piwik::getCurrentUserLogin(),
+            'emailAddress' => Piwik::getCurrentUserEmail()
+        ));
+        $email->safeSend();
```

Comment:
For these two emails, can we specify the login of the created user? So the superuser knows if which one to look at if the user is not recognized.